tests/e2e_logging.py

```diff
@@ -434,6 +439,50 @@ def test_historical_query(network, args):
     return network
 
 
+@reqs.description("Read historical receipts")
+@reqs.supports_methods("log/private", "log/private/historical_receipt")
+def test_historical_receipts(network, args):
+    if args.consensus == "bft":
+        LOG.warning("Skipping historical queries in BFT")
+        return network
+
+    if args.package == "liblogging":
+        node, _ = network.find_primary()
```

Comment:
Could we test this on all nodes in the network?

Reply:
Yes